R/net.inputs.R

```diff
@@ -354,12 +356,16 @@ param_random <- function(values, prob = NULL) {
 #' # Parameter model with deterministic and random parameters
 #' param <- param.net(inf.prob = 0.3, random.params = my_randoms)
 #'
+#' # A warning is produced if a random parameter overwrite a deterministic one
+#'
```

Comment:
@AdrienLeGuillou did you mean to include an example here? Also, let's edit the above text to say "...random parameter overwrites a fixed parameter."

Reply:
In this example, `acte.rate` is a random parameter but is also defined in the default `param.net` so a warning actually occurs. 

Maybe I should in this example use something else than `act.rate` to give an example without warnings and/or another one where I explicitly define a parameter as random and fixed